packages/story-editor/src/app/media/utils/useDetectVideoHasAudio.js

```diff
@@ -55,7 +55,7 @@ function useDetectVideoHasAudio({ updateMediaElement }) {
         return;
       }
       try {
-        const hasAudio = await hasVideoGotAudio(src);
+        const hasAudio = hasVideoGotAudio(await preloadVideo(src));
```

Comment:
I find this more readable. 

```
 const video = await preloadVideo(src);
 const hasAudio = hasVideoGotAudio(video);
```

Reply:
Really indifferent to me, both are equally readable to me. Personal preference I suppose 🙂

Reply:
Don't we also need to seek the video here as well?